src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/courtsentencing/CourtAppearanceAllMappingDto.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.courtsentencing
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.Size
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Court appearance mapping")
+data class CourtAppearanceAllMappingDto(
```

Comment:
extra dot required to enable one call from the update service for creating appearances and charges